src/pages/Petitions/index.tsx

```diff
@@ -62,6 +62,16 @@ const Petitions = (): JSX.Element => {
       search: new URLSearchParams(newSearchParams).toString(),
     })
   }
+  const handlePage = () => {
+    const newSearchParams = {
+      ...queryParams,
+      page: 1,
+    }
+    navigate({
+      pathname: '/petitions',
+      search: new URLSearchParams(newSearchParams).toString(),
+    })
+  }
```

Comment:
```suggestion
  const setInitialState = () => {
    setSortSelected('createdAt,desc')
    setCategorySelected(0)
    const newSearchParams = {
      ...queryParams,
      sort: 'createdAt,desc',
      categoryId: 0,
      page: 1,
    }
    navigate({
      pathname: '/petitions',
      search: new URLSearchParams(newSearchParams).toString(),
    })
  }
```

최신순/추천순
카테고리 분류도
초기화 될 수 있게 바꾸면 좋을 것 같고

handlePage 보단
setInitialState 를 사용하는 게 좋을 것 같아요